query/common.d.ts

```diff
@@ -269,6 +277,39 @@ export interface Bindings extends Iterable<[RDF.Variable, RDF.Term]> {
   ) => Bindings;
 }
 
+export interface Paths extends Iterable<Bindings> {
+  type: 'paths';
+  /**
+   * The number of variable-value pairs.
+   */
+  size: number;
+  /**
+   * Check if a binding exist for the given variable.
+   * @param key A variable term or string. If it is a string, no `?` prefix must be given.
+   */
+  has: (key: number) => boolean;
+  /**
+   * Obtain the binding value for the given variable.
+   * @param key A variable term or string. If it is a string, no `?` prefix must be given.
+   */
+  get: (key: RDF.Variable | string) => RDF.Term | undefined;
+  /**
+   * Iterator over all variable-value pairs.
+   */
+  [Symbol.iterator]: () => Iterator<Bindings>;
+  /**
+   * Return a list of bindings from the Path object
+   */
+  nodes: () => Bindings[];
+  /**
+   * Iterate over all variable-value pairs.
+   * @param fn A callback that is called for each variable-value pair
+   *           with value as first argument, and variable as second argument.
```

Comment:
I don't think this doc is right; instead the callback takes one Bindings argument.